packages/arb-ts/src/lib/message/L1ToL2Message.ts

```diff
@@ -183,6 +183,16 @@ export class L1TransactionReceipt implements TransactionReceipt {
     )
   }
 
+  public async looksLikeEthDeposit(
```

Comment:
Maybe add a comment explaining that this looks like a direct call to a specific function